seacrowd/sea_datasets/hse_thai/hse_thai.py

```diff
@@ -0,0 +1,139 @@
+import csv
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+    @misc{rtatman2017hse_thai,
+        author = {Rachel Tatman},
+        title = {HSE Thai Corpus},
+        howpublished = {\\url{https://www.kaggle.com/datasets/rtatman/hse-thai-corpus}},
+        note = {Accessed: 2023-11-22}
+    }
```

Comment:
Hmm, I know that the source is from Kaggle, but I think the original authors of the dataset are different. I traced it a bit and it led me to this: http://web-corpora.net/ThaiCorpus/search/. Maybe it's better to cite this website and the mentioned authors instead?

Reply:
Let me take a look into it later!